nautobot/core/utils/lookup.py

```diff
@@ -224,8 +224,9 @@ def get_created_and_last_updated_usernames_for_model(instance):
     created_by = None
     last_updated_by = None
     try:
-        created_by_record = object_change_records.get(action=ObjectChangeActionChoices.ACTION_CREATE)
-        created_by = created_by_record.user_name
+        created_by_record = object_change_records.filter(action=ObjectChangeActionChoices.ACTION_CREATE).first()
```

Comment:
With object-change factory data present, it's possible for there to be more than one CREATE record for a given object.